### PR TITLE
sidevm: Fix fd leak if the guest doesn't read input queries

### PR DESCRIPTION
**Pull request by kvinwang**

### CODE REVIEW

File: crates/pink/sidevm/host-runtime/src/env.rs [hunk 1]
```diff
@@ -213,18 +213,27 @@ impl Env {
...
+            let reply_tx = reply_tx?;
             let query = QueryRequest {
                 origin,
                 payload,
-                reply_tx: reply_tx?,
+                reply_tx,
             };
-            tx.send(query.encode()).await?;
+            let result = tx.send(query.encode()).await;
+            if result.is_err() {
```
Comment (h4x3rotab) [inline crates/pink/sidevm/host-runtime/src/env.rs:230]: Most of the case `result` could be error if the queue is full of unprocessed messages I guess? But we cannot really do much besides setting a timeout for reading the response. But at least, this doesn't cost us a lot. Is my understanding correct?

Comment (kvinwang) [reply crates/pink/sidevm/host-runtime/src/env.rs:230]: Exactly. It doesn't cost us a lot, at most the channel cap number of fds. It happens when the guest is stuck somewhere. Not a big problem.

Comment (h4x3rotab) [reply crates/pink/sidevm/host-runtime/src/env.rs:230]: Cool. I'm good with the fix

**Review by h4x3rotab — APPROVED** [review]
LGTM

**Pull request merged**